src/parquet/column/properties.h

```diff
@@ -161,26 +168,37 @@ class PARQUET_EXPORT WriterProperties {
       return this->compression(path->ToDotString(), codec);
     }
 
+    Builder* statistics(bool enable = true) {
+      default_column_settings_.collect_statistics = enable;
+      return this;
+    }
```

Comment:
Can you make enable not an optional argument? Seeing `...->statistics()` or `statistics('foo.bar.baz')` might not make it clear what's going on